std/string.d

```diff
@@ -3760,7 +3754,7 @@ auto leftJustifier(Range)(Range r, size_t width, dchar fillChar = ' ')
 
             static if (isForwardRange!Range)
             {
-                @property typeof(this) save()
+                @property typeof(this) save() return scope
```

Comment:
Wait, I thought `return scope` on the `this` reference only made sense if the returned value has indirections. This doesn't return a value with indirections.